src/Core/Usecase/Message/ReceiveMessage.php

```diff
@@ -44,7 +47,8 @@ public function setPostRepository(CreateRepository $postRepo)
 		$this->postRepo = $postRepo;
 		return $this;
 	}
-
+	protected $targeted_survey_state_repo;
+	protected $form_attr_repo;
```

Comment:
Ugh, sorry if these underscores vs CamelCases are my fault. But we should probably clean that up.

Reply:
fixed thanks